packages/frontend/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -2993,142 +2993,181 @@
+	"evaluation.listRuns.noRuns.description": "Run a test to see the results here",
+	"evaluation.listRuns.deleteRuns": "No runs to delete | Delete {count} run | Delete {count} runs",
+	"evaluation.listRuns.noRuns.button": "Run Test",
+	"evaluation.listRuns.error.testCasesNotFound": "No matching rows in dataset{description}",
+	"evaluation.listRuns.error.testCasesNotFound.description": "Check any filters or limits set in the evaluation trigger",
+	"evaluation.listRuns.error.executionInterrupted": "Test run was interrupted",
+	"evaluation.listRuns.error.unknownError": "Execution error{description}",
+	"evaluation.listRuns.error.unknownError.description": "Click for more details",
+	"evaluation.listRuns.error.evaluationTriggerNotFound": "Evaluation trigger missing",
+	"evaluation.listRuns.error.evaluationTriggerNotConfigured": "Evaluation trigger is not configured",
+	"evaluation.listRuns.error.evaluationTriggerDisabled": "Evaluation trigger is disabled",
+	"evaluation.listRuns.error.setOutputsNodeNotFound": "No 'Set outputs' node in workflow",
+	"evaluation.listRuns.error.setOutputsNodeNotConfigured": "'Set outputs' node is not configured",
+	"evaluation.listRuns.error.setMetricsNodeNotFound": "No 'Set metrics' node in workflow",
+	"evaluation.listRuns.error.setMetricsNodeNotConfigured": "'Set metrics' node is not configured",
+	"evaluation.listRuns.error.cantFetchTestCases": "Couldn’t fetch test cases{description}",
+	"evaluation.listRuns.error.cantFetchTestCases.description": "Check the Google Sheet setup in the evaluation trigger",
+	"evaluation.runDetail.ranAt": "Ran at",
+	"evaluation.runDetail.testCase": "Test case",
+	"evaluation.runDetail.testCase.id": "Test case ID",
+	"evaluation.runDetail.testCase.status": "Test case status",
+	"evaluation.runDetail.totalCases": "Total cases",
+	"evaluation.runDetail.error.mockedNodeMissing": "Output for a mocked node does not exist in benchmark execution.{link}.",
+	"evaluation.runDetail.error.mockedNodeMissing.solution": "Fix test configuration",
+	"evaluation.runDetail.error.executionFailed": "Failed to execute workflow",
+	"evaluation.runDetail.error.executionFailed.solution": "View execution",
+	"evaluation.runDetail.error.datasetTriggerNotFound": "Dataset trigger does not exist in the workflow.{link}.",
+	"evaluation.runDetail.error.datasetTriggerNotFound.solution": "View workflow",
+	"evaluation.runDetail.error.invalidMetrics": "Evaluation metrics node returned invalid metrics. Only numeric values are expected. View workflow. {link}.",
+	"evaluation.runDetail.error.invalidMetrics.solution": "View workflow",
+	"evaluation.runDetail.error.unknownError": "An unknown error occurred",
+	"evaluation.runDetail.error.unknownError.solution": "View execution",
+	"evaluation.runDetail.error.noMetricsCollected": "No 'Set metrics' node executed",
+	"evaluation.runDetail.error.partialCasesFailed": "Finished with errors",
+	"evaluation.runTest": "Run Test",
+	"evaluation.cancelTestRun": "Cancel Test Run",
```

Comment:
this does not seem used

Reply:
There are a number that are not used, for example `evaluation.edit.*`... Will go through and remove.